src/AssetTransferApi.spec.ts

```diff
@@ -336,6 +336,22 @@ describe('AssetTransferAPI', () => {
 					moonriverAssetsApi.registry
 				);
 
+				expect(assetCallType).toEqual('Reserve');
+			});
+		});
+		describe('ParaToRelay', () => {
+			it('Should correctly return XTokens', () => {
```

Comment:
should we have`return Reserve` here in the `it` or do we mean to add `xTokens` as a new fetchCallType?

Reply:
Ahh yes, you're right it should return `Reserve`, good catch